### PR TITLE
Misc fixes.

### PR DESCRIPTION
**Pull request by aomarks**

### CODE REVIEW

File: iron-doc-nav.html [hunk 1]
```diff
@@ -110,14 +110,24 @@
```
Comment (samuelli) [inline iron-doc-nav.html:110]: Won't let me add a comment further up.

Just noticed that on line 60, section is redefined again. Can you change this to use another variable name?

File: iron-doc-nav.html [hunk 2]
```diff
@@ -110,14 +110,24 @@
...
           ];
+
+          var pruned = [];
+          for (var i = 0; i < sections.length; i++) {
+            if (sections[i].items.length) {
+              pruned.push(sections[i]);
+            }
+          }
+
+          this._sections = pruned;
```
Comment (samuelli) [inline iron-doc-nav.html]: Another technique I've used is to use data binding.

```
<section hidden$="[[!list.length]]">
...
```

Would that be more suitable than this pruning approach?

File: iron-doc-nav.html (hunk 1 above)
Comment (aomarks) [reply iron-doc-nav.html:110]: Oops! Done (back to item).

File: iron-doc-nav.html (hunk 2 above)
Comment (aomarks) [reply iron-doc-nav.html]: Yeah that's way simpler :D

**Review by samuelli — APPROVED** [review]

**Pull request merged**